server/src/main/java/org/elasticsearch/cluster/block/ClusterBlock.java

```diff
@@ -142,7 +144,12 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeVInt(id);
         out.writeOptionalString(uuid);
         out.writeString(description);
-        out.writeEnumSet(levels);
+        if (out.getTransportVersion().onOrAfter(TransportVersions.NEW_REFRESH_CLUSTER_BLOCK)) {
+            out.writeEnumSet(levels);
+        } else {
+            // do not send ClusterBlockLevel.REFRESH to old nodes
+            out.writeEnumSet(filterLevels(levels, level -> ClusterBlockLevel.REFRESH.equals(level) == false));
```

Comment:
No, there is no test that creates a new index with a REFRESH block in a mixed cluster version. There is also a condition to NOT add the block if a node of the cluster is in a old version.

With this in mind, I think it should work as it is but I prefer to add the bwc support for correctness, just in case.

Reply:
Oh I see so it's ineffective if there's any older node. OK sounds good to me.